course/models.py

```diff
@@ -27,9 +27,11 @@ def get_default_academic_year():
 
 class Course(models.Model):
 
+    color = models.CharField(max_length = 7, null = True, blank=True)
```

Comment:
Color can't be a model parameter. Every student has a different computed value for color attribute.